stage2/01-sys-tweaks/00-patches/series

```diff
@@ -1,4 +1,3 @@
 01-useradd.diff
 04-inputrc.diff
 05-path.diff
-07-resize-init.diff
```

Comment:
Why did you remove this? That's important for auto-resizing the disk on first boot.

Reply:
Hmm, you're right, it's unnecessary.